org.sdrangel.SDRangel.json

```diff
@@ -0,0 +1,525 @@
+{
+  "app-id": "org.sdrangel.SDRangel",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.15-21.08",
+  "sdk": "org.kde.Sdk",
+  "command": "sdrangel",
+  "rename-desktop-file": "sdrangel.desktop",
+  "rename-icon": "sdrangel_icon",
+  "copy-icon": true,
+  "finish-args": [
+    "--filesystem=xdg-documents",
+    "--device=all",
+    "--share=network",
+    "--share=ipc",
+    "--socket=pulseaudio",
+    "--socket=x11",
+    "--socket=wayland",
```

Comment:
```suggestion
```

Reply:
Thanks. I remember that there was some problem when running on Wayland-enabled desktop with the Wayland socket disabled, but I see that QT_QPA_PLATFORM is set to `xcb` (because of a Wayland-only bug), so it probably should be fine to remove the Wayland socket, but I need to test this first.

Reply:
THere are a few apps I maintain that are x11 only (with Qt) there is no problem.

wayland is the problem

(I run GNOME + Mutter on top of Wayland)